top-k-frequent-elements/EcoFriendlyAppleSu.kt

```diff
@@ -0,0 +1,26 @@
+package leetcode_study
+
+/**
+ * 주어진 숫자들에서 빈도 수가 가장 큰 k 개의 숫자를 구하는 문제. map 자료구조를 사용해 해결
+ * 시간 복잡도 : O(n)
```

Comment:
아래 설명을 보면 이 곳은 오타겠지요?

```suggestion
 * 시간 복잡도 : O(nlogn)
```

Reply:
가장 큰 k 개의 숫자만 필요한데 굳이 전체 숫자를 정렬할 필요가 있을지 고민해보시면 더 효율적인 알고리즘을 얻으실 수 있을 것 같아요. 현재 풀이도 훌륭하지만 빨리 제출해주셔서 드리는 피드백입니다.